frog/imports/client/SingleActivity/Finish.js

```diff
@@ -37,11 +37,13 @@ function Finish(
       </IconButton>
       <Typography variant="h3" component="h2" className={classes.padded_text}>
         You're all set, please share this link with the participants:{' '}
-        <Link to={'/' + url.public}>{BASE_URL + '/' + url.public}</Link>
```

Comment:
Removing the link tag would not allow the user to click on the link

Reply:
This was intentional, teachers should not click the student URL.